plantuml-mode.el

```diff
@@ -111,12 +112,24 @@
   :type '(repeat string)
   :group 'plantuml)
 
+(defcustom plantuml-server-url "https://www.plantuml.com/plantuml"
+  "The base URL of the PlantUML server."
+  :type 'string
+  :group 'plantuml)
+
+(defcustom plantuml-default-exec-mode 'server
```

Comment:
The documentation update notes that `'jar` is the default mode, isn't this setting the default to `'server` instead?